src/radarChart.ts

```diff
@@ -313,7 +243,7 @@ module powerbi.extensibility.visual {
 
             for (let i: number = 0, iLen: number = categoryValues.length; i < iLen; i++) {
                 labelsData.labelPoints.push({
-                    text: categoryValues[i],
+                    text: categoryValues[i].toString(),
```

Comment:
Is it possible that `categoryValues[i]` might be `null`? I suppose that it'd be better to use type casting: `categoryValues[i] as string`